src/serviceconnector-passwordless/setup.py

```diff
@@ -15,7 +15,7 @@
     logger.warn("Wheel is not available, disabling bdist_wheel hook")
 
 
-VERSION = '3.0.0'
+VERSION = '3.0.1'
 try:
     from azext_serviceconnector_passwordless.config import VERSION
```

Comment:
It seems there is a dup VERSION from config.py. 
We recommend setting VERSION in setup.py. Please delete the dup one from config.py